activities/Gridpaint.activity/css/activity.css

```diff
@@ -28,7 +28,7 @@
   visibility: hidden;
 }
 
-.tutorial-title {
+/* .tutorial-title {
```

Comment:
Do not keep commented code

Reply:
hey @llaske i have updated the code for the grid activity pls check